run-tests.sh

```diff
@@ -17,4 +17,9 @@ case "$(uname)" in
         ;;
 esac
 
+if [ "$1" = "--ci" ]
+then shift
+else set -- "$@" -m 'not invasive'
```

Comment:
what about making it just not be ran by default, unless e.g. `--ci` or `--invasive` passed to pytest? some possible setups for that are at https://stackoverflow.com/questions/33084190/default-skip-test-unless-command-line-parameter-present-in-py-test  - otherwise folks who just run `python -m pytest` might suffer